tensorflow/docs_src/install/install_sources.md

```diff
@@ -25,8 +25,10 @@ like to try to build TensorFlow on Windows anyway, use either of the
 following:
 
 *   [Bazel on Windows](https://bazel.build/versions/master/docs/windows.html)
-*   [TensorFlow CMake build](https://github.com/tensorflow/tensorflow/tree/r0.12/tensorflow/contrib/cmake)
+*   [TensorFlow CMake build](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/cmake)
```

Comment:
Actually, should this point to `master` or the current release version? (Though I guess it doesn't matter a whole lot)

Reply:
(I'm pretty sure) this is handled automatically by the site-generation tools.